lib/util.js

```diff
@@ -76,14 +76,23 @@ exports.generateStreamArrayForLevels = function (lvls, stream) {
  * @return {Array}
  */
 exports.getDefaultStreams = function () {
-  return (env.get('TKE_LOG_SILENT') === 'true') ? [] : [{
+  var logStreams = [{
     stream: process.stdout,
     level: exports.getLogLevel()
   }, {
     stream: process.stderr,
     level: 'warn'
-  }, {
-    level: 'warn',
-    stream: require('./mongo-stream')
   }];
+
+  if (env.get('TKE_LOG_SILENT') === 'true') {
+    return [];
+  }
+  else if (env.get('FH_MONGODB_CONN_URL')) {
```

Comment:
This can just be an _if_, no need for else unless I'm misunderstanding seomthing?